app/lib/style/style.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flutter/material.dart';
+
+const SmallTextSize = 14.0;
+const BodyTextSize = 16.0;
+const MediumTextSize = 20.0;
+const LargeTextSize = 23.0;
+
+const SelectedItemColor = Colors.green;
+
+const BottomBorderStyle = BorderSide(
+  // Add more properties if needed
```

Comment:
Unnecessary comments (no need to remove now)

Reply:
Will remove in my future experimentation